### PR TITLE
:sparkles: save readme button

### PR DESCRIPTION
**Pull request by bpetetot**
**Description**
Add the save button of the readme documentation field. (only GUI part)
Call the endpoint `POST /api/readme`

### CODE REVIEW

**Comment by bpetetot** [comment]
Can you just check about how I am using saga in this simple case ?
To be sure that I'm doing it in the right way. :)

File: src/gui/sagas/documentation/readme.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { select } from 'redux-saga/effects'
+import { readme } from '../../redux/documentation'
+
+export function* saveReadme() {
+  const state = yield select(readme.get())
+
+  fetch('/api/readme', { method: 'POST', body: state })
```
Comment (fabienjuif) [inline src/gui/sagas/documentation/readme.js]: Look at `yield call` so test are easier to write :)

File: src/gui/screens/documentation/actions/actions.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import SaveButton from '../../../components/button'
+
+const Actions = ({ onSave }) => {
+  return (
+    <div>
```
Comment (fabienjuif) [inline src/gui/screens/documentation/actions/actions.jsx]: Why do you wrap it to a div ?

File: src/gui/screens/documentation/actions/actions.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      <SaveButton label="Save" onClick={onSave} />
+    </div>
+  )
+}
+
+Actions.propTypes = {
+  onSave: PropTypes.func.isRequired,
+}
+
+export default Actions
```
Comment (fabienjuif) [inline src/gui/screens/documentation/actions/actions.jsx]: I think this a good practice to always use `onlyUpdateForPropTypes` from `recompose`

https://github.com/Zenika/react-workbench/blob/master/src/gui/screens/workbench/state/state.jsx#L32

**Review by fabienjuif — CHANGES_REQUESTED** [review]
Some comments.
If you use sagas effects, tests are easier to write, espacially with trampss-redux-saga-tester :)

**Pull request merged**